motor_controller/test/unittest_motor_controller.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import rosunit
+from motor_controller.motor_controller import MotorController
+from unittest import TestCase, mock
+
+
+#@patch('motor_controller.motor_controller.serial.Serial')
```

Comment:
Why is this commented?

Reply:
This was left from early stages when me and you tried to get the mock working.

Reply:
Removed line 7.